docs/Privacy_policy.md

```diff
@@ -1,3 +1,67 @@
+
+# Collecting and Using Your Personal Data
+## Types of Data Collected
+### Personal Data
+While using Our Service, We may ask You to provide Us with certain personally identifiable information that can be used to contact or identify You. Personally identifiable information may include, but is not limited to:
+* PSID
+* Messages
+
+### PSID
+PSID is an ID assigned to you when you first initiate a conversation with the bot, which is permanent, and specific to the Bot's Facebook Page MovieBot. This is stored along with the conversation messages to identify you.
+
+### Messasges
+Messages are all messages exchanged with the bot, both sent by you and the bot.
+
+# Your Pesonal Data
+## Use of Your Personal Data
+The Bot will use Personal Data for the purpose of improving the reccomendations, making them more accurate.
+
+## Retention of Your Personal Data
+The organisation will retain Your Personal Data only for as long as is necessary for the purposes set out in this Privacy Policy. We will retain and use Your Personal Data to the extent necessary to comply with our legal obligations (for example, if we are required to retain your data to comply with applicable laws), resolve disputes, and enforce our legal agreements and policies.
+
+The organisation will also retain Usage Data for internal analysis purposes. Usage Data is generally retained for a shorter period of time, except when this data is used to strengthen the security or to improve the functionality of Our Service, or We are legally obligated to retain this data for longer time periods.
+
+If at any point you want to delete your data, you may do so by sending "/delete" to the bot
+
+## Transfer of Your Personal Daata
+Your information, including Personal Data, is processed at the organisation's operating offices and in any other places where the parties involved in the processing are located. It means that this information may be transferred to — and maintained on — computers located outside of Your state, province, country or other governmental jurisdiction where the data protection laws may differ than those from Your jurisdiction.
+
+Your consent to this Privacy Policy followed by Your submission of such information represents Your agreement to that transfer.
+
+The organisation will take all steps reasonably necessary to ensure that Your data is treated securely and in accordance with this Privacy Policy and no transfer of Your Personal Data will take place to an organization or a country unless there are adequate controls in place including the security of Your data and other personal information.
+
+## Disclosure of Your Personal Data
+### Business Transactions
+If the organisation is involved in a merger, acquisition or asset sale, Your Personal Data may be transferred. We will provide notice before Your Personal Data is transferred and becomes subject to a different Privacy Policy.
```

Comment:
"If the Organisation undergoes changes in name, ownership, or any significant changes in its structure, Your Personal Data may be transferred."